### ISSUE TITLE
No character limit in question

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#494]
Questions should not have 20 character limit.

**Comment by roschaefer** [comment]
we implemented that in #317

**Comment by roschaefer** [comment]
changed your mind?

**Comment by drjakob** [comment]
Other Editor changed his mind. Conflict of interest:  Sensorstory needs longer Questions, but chatfuel is limited to 20 characters. Sorry about this.

**Comment by roschaefer** [comment]
Is the editor aware, that questions longer than 20 characters will not be visible in chatfuel?

**Comment by drjakob** [comment]
Yes. But editorial decision in this project: some tcs for chatfuel only, others for sensorstory only. No mixed channel tcs.

**Comment by drjakob** [comment]
So for sensorstory tcs i see the adbantage of longer questions.

**Comment by roschaefer** [comment]
Ok, will @DavidLichtenberger & friends ever implement the RESI feature for the frontend? :wink: 

It's OK for me to drop the constraint.

**Comment by drjakob** [comment]
Never heard of this feature.
